1.9.0 Breaking Change - Less importing no longer working

# 🐛 bug report

1.8.1 was importing less without issue. Now I am getting:

PROJECT_DIR/www/css/main.less: Cannot find module 'uploader.less' from 'PROJECT_DIR/www/node_modules/rsuite/styles'

```
~/c/g/s/g/t/t/www ❯❯❯ cat css/main.less                                                                                                                                                                                                                     
@import 'rsuite/styles/rsuite.less'; //The default style file.


~/c/g/s/g/t/t/www ❯❯❯ ls -la node_modules/rsuite/styles/uploader.less                                                                                                                                                                                       feature/www ✭ ✖ ✱ ◼
-rw-r--r--  1 raja  staff  10925 14 Jun 07:46 node_modules/rsuite/styles/uploader.less

```




## 🎛 Configuration (.babelrc, package.json, cli command)

package.json:
```js
{
  "name": "thehiringco",
  "version": "1.0.0",
  "description": "thehiringco react frontend",
  "main": "index.html",
  "private": true,
  "scripts": {
    "start": "parcel index.html",
    "clean": "rm -rf .cache && rm -rf dist"
  },
  "dependencies": {
    "apollo-cache-inmemory": "^1.2.3",
    "apollo-client": "^2.3.3",
    "apollo-link": "^1.2.2",
    "apollo-link-error": "^1.1.0",
    "apollo-link-http": "^1.5.4",
    "apollo-link-state": "^0.4.1",
    "babel-plugin-transform-class-properties": "^6.24.1",
    "babel-preset-env": "^1.7.0",
    "babel-preset-es2015": "^6.24.1",
    "babel-preset-react": "^6.24.1",
    "babel-preset-stage-2": "^6.24.1",
    "dom-lib": "^1.0.0",
    "graphql": "^0.13.2",
    "graphql-tag": "^2.9.2",
    "howler": "^2.0.12",
    "less": "2.7.3",
    "lodash": "^4.17.10",
    "parcel-bundler": "^1.9.0",
    "react": "^16.4.0",
    "react-apollo": "^2.1.4",
    "react-dom": "^16.4.0",
    "react-router-dom": "^4.3.1",
    "router": "^1.3.2",
    "rsuite": "^3.0.8"
  },
  "devDependencies": {
    "babel-eslint": "^8.2.3",
    "eslint": "^4.19.1",
    "eslint-config-airbnb": "^16.1.0",
    "eslint-plugin-flowtype": "^2.49.3",
    "eslint-plugin-import": "^2.12.0",
    "eslint-plugin-jsx-a11y": "^6.0.3",
    "eslint-plugin-node": "^6.0.1",
    "eslint-plugin-promise": "^3.8.0",
    "eslint-plugin-react": "^7.9.1",
    "eslint-plugin-standard": "^3.1.0"
  }
}
```

## 🤔 Expected Behavior

Should bundle fine as it did in 1.8.1

## 😯 Current Behavior

Cannot find less file that exists

## 🌍 Your Environment

<!--- Include as many relevant details about the environment you experienced the bug in -->

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           | 1.9.0
| Node             | 10.4.1
| npm/Yarn         | 1.7.0
| Operating System | macOS

--- a/src/assets/LESSAsset.js
+++ b/src/assets/LESSAsset.js
@@ -4,6 +4,7 @@ const promisify = require('../utils/promisify');
 const Resolver = require('../Resolver');
 const fs = require('../utils/fs');
 const path = require('path');
+const parseCSSImport = require('../utils/parseCSSImport');
 
 class LESSAsset extends Asset {
   constructor(name, options) {
@@ -80,6 +81,7 @@ function getFileManager(less, options) {
     }
 
     async loadFile(filename, currentDirectory) {
+      filename = parseCSSImport(filename);
       let resolved = await resolver.resolve(
         filename,
         path.join(currentDirectory, 'index')

--- a/src/assets/SASSAsset.js
+++ b/src/assets/SASSAsset.js
@@ -4,6 +4,7 @@ const promisify = require('../utils/promisify');
 const path = require('path');
 const os = require('os');
 const Resolver = require('../Resolver');
+const parseCSSImport = require('../utils/parseCSSImport');
 
 class SASSAsset extends Asset {
   constructor(name, options) {
@@ -50,11 +51,7 @@ class SASSAsset extends Asset {
       ? opts.importer
       : [opts.importer];
     opts.importer.push((url, prev, done) => {
-      if (!/^(~|\.\/|\/)/.test(url)) {
-        url = './' + url;
-      } else if (!/^(~\/|\.\/|\/)/.test(url)) {
-        url = url.substring(1);
-      }
+      url = parseCSSImport(url);
       resolver
         .resolve(url, prev === 'stdin' ? this.name : prev)
         .then(resolved => resolved.path)

--- a/src/utils/parseCSSImport.js
+++ b/src/utils/parseCSSImport.js
@@ -1,0 +1,10 @@
+function parseCSSImport(url) {
+  if (!/^(~|\.\/|\/)/.test(url)) {
+    url = './' + url;
+  } else if (!/^(~\/|\.\/|\/)/.test(url)) {
+    url = url.substring(1);
+  }
+  return url;
+}
+
+module.exports = parseCSSImport;

--- a/test/integration/less-import/base.less
+++ b/test/integration/less-import/base.less
@@ -1,3 +1,5 @@
+@import '~external-less/a.less';
+
 @base: #f938ab;
 
 .base {

